modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFReaderTest.java

```diff
@@ -18,15 +18,16 @@
 
 import it.geosolutions.jaiext.JAIExt;
 import it.geosolutions.jaiext.range.NoDataContainer;
+import ucar.nc2.dataset.NetcdfDataset;
 
-import java.awt.Dimension;
-import java.awt.Rectangle;
+import java.awt.*;
```

Comment:
Too many classes imported here java.awt is a big package. Perhaps change your IDE organise-imports settings? I think Eclipse avoids * imports until you have 99 from one package.